srcpkgs/gsfonts/template

```diff
@@ -1,20 +1,42 @@
 # Template file for 'gsfonts'
 pkgname=gsfonts
-version=8.11
-revision=7
-wrksrc=fonts
-makedepends="font-util font-misc-misc"
-depends="${makedepends}"
-short_desc="Ghostscript standard Type1 fonts"
+version=20200910
+revision=1
+wrksrc="urw-base35-fonts-${version}"
+depends="font-util"
+short_desc="URW+ base35 fonts"
 maintainer="Orphaned <orphan@voidlinux.org>"
-license="GPL-2.0-or-later"
-homepage="http://sourceforge.net/projects/gs-fonts/"
-distfiles="${SOURCEFORGE_SITE}/ghostscript/ghostscript-fonts-std-${version}.tar.gz"
-distfiles="${SOURCEFORGE_SITE}/gs-fonts/ghostscript-fonts-std-${version}.tar.gz"
-checksum=0eb6f356119f2e49b2563210852e17f57f9dcc5755f350a69a46a0d641a0c401
-font_dirs="/usr/share/fonts/Type1"
+license="AGPL-3.0-or-later"
+homepage="https://github.com/ArtifexSoftware/urw-base35-fonts"
+distfiles="https://github.com/ArtifexSoftware/urw-base35-fonts/archive/refs/tags/${version}.tar.gz"
+checksum=e0d9b7f11885fdfdc4987f06b2aa0565ad2a4af52b22e5ebf79e1a98abd0ae2f
+font_dirs="/usr/share/fonts/OTF/
+ /usr/share/fonts/TTF
+ /usr/share/fonts/Type1"
 
 do_install() {
-	vmkdir usr/share/fonts/Type1
-	install -m644 ${wrksrc}/*.[ap]f[bm] ${DESTDIR}/usr/share/fonts/Type1
+	for f in ${wrksrc}/fonts/*.otf; do
+		vinstall $f 644 usr/share/fonts/OTF/
+	done
+
+	for f in ${wrksrc}/fonts/*.ttf; do
+		vinstall $f 644 usr/share/fonts/TTF/
+	done
+
+	for f in ${wrksrc}/fonts/*.{afm,t1}; do
```

Comment:
```suggestion
	for f in ${wrksrc}/fonts/*.{[ap]f[bm],t1}; do
```

Reply:
The archive only contains .afm and .t1 files. No .pfm or .pbm.

Reply:
ah ok, it seems to have had those previously